Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM gcr.io/distroless/static:debug-nonroot
+WORKDIR /
+
+COPY manager manager
+EXPOSE 8080
```

Comment:
Update this to build the PO manager executable using downstream images? This can be done in a follow-up though.